examples/utils/oauth.js

```diff
@@ -1,5 +1,5 @@
 // Authenticate with Genesys Cloud
-function authenticate(client, pcEnvironment) {
+function authenticate(client, pcEnvironment, state) {
```

Comment:
Just need to assert all the calls are updated here.
Also, need to make sure we deploy these utils in all instances.  I do think it's the case, but, want to confirm.

Reply:
lgtm!